.gitignore

```diff
@@ -381,6 +381,9 @@ installer/test/testpackages/
 # Local temp storage
 temp/
 
+# Local per-user storage
+.user/
```

Comment:
(FYI for @EricJohnson327... I think you talked about having a similar need for package caching, etc.)